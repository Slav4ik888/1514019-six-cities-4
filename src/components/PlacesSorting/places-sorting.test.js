import React from 'react';
import renderer from 'react-test-renderer';

import PlacesSorting from './places-sorting.jsx';

const child = <div />;

describe(`Snapshot <PlacesSorting/>`, () => {
  it(`Render <PlacesSorting/>`, () => {
    const tree = renderer.create(
        <PlacesSorting>
          {child}
        </PlacesSorting>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

});
