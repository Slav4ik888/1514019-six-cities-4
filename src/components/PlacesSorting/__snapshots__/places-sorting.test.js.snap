// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot <PlacesSorting/> Render <PlacesSorting/> 1`] = `
<form
  action="#"
  className="places__sorting"
  method="get"
>
  <span
    className="places__sorting-caption"
  >
    Sort by 
  </span>
  <div />
</form>
`;
